pkg/sharing/replicator.go

```diff
@@ -237,6 +237,9 @@ func (s *Sharing) callChangesFeed(inst *instance.Instance, since string) (*Chang
 		if !ok {
 			continue
 		}
+		if _, ok = info["binary"]; ok {
+			continue
```

Comment:
Shouldn't it be `!ok` ?

Reply:
No, I really meant `ok`. We are here in the replicator (the first phase of synchronization of a sharing), and binaries are managed in the second phase ("the upload"), so we want to skip them here.